test_runner/src/main/kotlin/ftl/util/ObjPath.kt

```diff
@@ -16,12 +17,16 @@ data class ObjPath(
             val objName = parsed.getName(0).toString()
             val shardName = parsed.getName(1).toString()
             val deviceName = parsed.getName(2).toString()
+            val filePathName = if (parsed.nameCount > 4) {
+                parsed.subpath(3, parsed.nameCount - 1).toString()
```

Comment:
What do you think of `parsed.parent.drop(3).joinToString("/")` as a way to simplify the `parsed.subpath` logic?